lib/rules/no-multi-spaces.js

```diff
@@ -12,7 +14,17 @@
 module.exports = function(context) {
 
     // the index of the last comment that was checked
-    var lastCommentIndex = 0;
+    var exceptions = { "Property": true },
+        exceptionCount = 1,
+        options = context.options[0],
+        lastCommentIndex = 0;
+
+    if (options && options.exceptions) {
+        Object.keys(options.exceptions).forEach(function (key) {
+            exceptions[key] = options.exceptions[key];
+        });
+        exceptionCount = Object.keys(exceptions).length;
```

Comment:
The goal is to check the type of the parent node only if there are exceptions. If there are no exceptions, it doesn't matter what the parent is, so there's no reason to traverse down the AST to find it. So it won't go away, I just need to make sure it's actually deleting keys rather than setting them to false. That way if someone overrides and removes `Property`, `exceptionCount` will be 0, and no traversal will occur.